api/handlers/containerhandler.py

```diff
@@ -64,7 +65,7 @@ class ContainerHandler(base.RequestHandler):
             'parent_storage': containerstorage.ProjectStorage(),
             'storage_schema_file': 'session.json',
             'payload_schema_file': 'session.json',
-            'list_projection': {'info': 0, 'analyses': 0},
+            'list_projection': {'info': 0, 'analyses': 0, 'subject.firstname': 0, 'subject.lastname': 0},
```

Comment:
It's only excluding on the `get_all` handler endpoint. I might be confused about what surprised you - did you expect it to be explicit in the container storage layer?